src/less/styles.less

```diff
@@ -165,19 +214,21 @@ a[name]:before {
 }
 
 // Headers
+h1, h2, h3 {
+  margin-bottom: 1.5rem;
+}
+
 h1 {
   font-weight: 600;
 }
 h2 {
-  font-weight: 600;
+  font-weight: 500;
 }
 h3 {
   font-weight: 400;
-  color: @nats-colour-blue-dark;
 }
```

Comment:
We need to have more of a difference between h2 and h3 - looking at the 'How to Contribute' doc, Contributing Code section.